src/Sulu/Bundle/SecurityBundle/Factory/UserFactoryInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\SecurityBundle\Factory;
+
+use Sulu\Component\Security\Authentication\UserInterface;
+
+interface UserFactoryInterface
+{
+    public function create(
+        string $username,
+        string $firstName,
+        string $lastName,
+        string $email,
+        string $locale,
+        string $password,
+        string $roleName = 'User'
```

Comment:
Same as above, would avoid to set the default value to something that might break if we change a value in the database.